src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.DateTime.corefx.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Diagnostics
+{
+    partial class Activity
+    {
+        private DateTime GetUtcNow()
```

Comment:
Place a comment that says in .NET Core (on Win8 or above) DateTime.UtcNow has good accuracy (typically 1usec), and thus can be used directly.